requirements.txt

```diff
@@ -1,6 +1,12 @@
+# basic dependencies
 numpy~=1.15
 periodictable~=1.5
+
+# PICMI
 picmistandard==0.0.16
+#   for development against an unreleased PICMI version, use:
+#picmistandard @ git+https://github.com/picmi-standard/picmi.git#subdirectory=PICMI_Python
```

Comment:
Needs pip 20.1+
```
python3 -m pip install -U pip
```

Reply:
This is presumably a fix to use when building WarpX locally. The `development` branch should always depend on a specific version of picmi, correct?

Reply:
Yes, correct.
This PR just adds a comment in `requirements.txt` how to change this if one wants to.

The background is that we are extending PICMI for 1D simulations at the moment, and we realized we did not yet document that workflow for local testing of an experimental/upcoming PICMI version.